.github/workflows/main.yml

```diff
@@ -16,11 +15,107 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v3
 
+      - name: Read Review Guidelines
+        id: read-guidelines
+        run: |
+          REVIEW_GUIDELINES=$(cat REVIEW_GUIDE.md)
+          echo "::set-output name=review_guidelines::$REVIEW_GUIDELINES"
+
       - name: AI Code Review Action
         uses: freeedcom/ai-codereviewer@v2.6.0
         with:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
           OPENAI_API_MODEL: "gpt-4" # Optional: defaults to "gpt-4"
-          exclude: "**/*.json, **/*.md" # Optional: exclude patterns separated by commas
-          post-if-error: 'true'
+          exclude: "**/*.json,**/*.md" # Optional: exclude patterns separated by commas
+# on:
+#   pull_request:
```

Comment:
The removal of `post-if-error` may prevent error messages from being posted. If this is not the intended behavior, consider keeping this line.